src/sessions/base-session-handler.ts

```diff
@@ -92,7 +96,10 @@ export default abstract class BaseSessionHandler {
     });
 
     session.on('terminated', this.onSessionTerminated.bind(this, session));
-    this.sdk.emit('sessionStarted', session);
+
+    if (!session.reinvite) {
```

Comment:
we want the reinvite sessions to silently replace the existing session so I can't think of a reason why we'd want to notify consumers about a reinvite session.